pkgs/development/tools/micronaut/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchzip, jdk, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "micronaut-${version}";
+  version = "1.2.1";
+  src = fetchzip {
+    url = "https://github.com/micronaut-projects/micronaut-core/releases/download/v1.2.1/micronaut-1.2.1.zip";
+    sha256 = "0lfl2hfakpdcfii3a3jr6kws731jamy4fb3dmlnj5ydk0zbnmk6r";
+  };
+  nativeBuildInputs = [makeWrapper];
+  installPhase = ''
+    rm bin/mn.bat
+    cp -r . $out
+    wrapProgram $out/bin/mn \
+      --prefix JAVA_HOME : ${jdk} 
+  '';
+    meta = with stdenv.lib; {
+      description = ''
+        A modern, JVM-based, full-stack framework for building modular, 
+        easily testable microservice and serverless applications.
+      '';
+      longDescription = ''
+        Reflection-based IoC frameworks load and cache reflection data for 
+        every single field, method, and constructor in your code, whereas with 
+        Micronaut, your application startup time and memory consumption are 
+        not bound to the size of your codebase.
+      '';
+      homepage = https://micronaut.io/;
+      license = licenses.mit;
```

Comment:
The LICENSE file says apache 2.0 (`asl20`)

Reply:
Sorry about that I must have been looking at a different results dir